src/addons/jest-matchers.js

```diff
@@ -18,6 +18,11 @@ expect.extend({
   },
 
   toHaveStatus (action, status) {
+    if (action instanceof Action === false) {
+      throw new TypeError("toHaveStatus expects an Action. Received " +
+                          (action ? "a " + action.constructor.name : action) + ".")
```

Comment:
You want to return null if no action on the second ternary return value? Might be easier to read by explicitly having `null` there?